_includes/components/2-col.html

```diff
@@ -1,4 +1,4 @@
-<article class="container {{ include.class }} page-content__prose">
+<div class="container container--mod {{ include.class }} page-content__prose">
```

Comment:
What does "mod" stand for? Could we pick a more semantically meaningful name so that its purpose is clearer?

I'm also still not entirely clear what issues there would be with using `page-content__prose`, since if the idea with the prose class is to apply general content styles like line-height, ideally that should be the only class we'd need.

Reply:
> What does "mod" stand for? Could we pick a more semantically meaningful name so that its purpose is clearer?

Yes, I was playing around with names and couldn't think of anything.

> I'm also still not entirely clear what issues there would be with using `page-content__prose`, since if the idea with the prose class is to apply general content styles like line-height, ideally that should be the only class we'd need.

When messing around with `page-content__prose`, I lost the bottom margin of the  `<h2>` element as well as the `margin-top` of the `<h3>`. Then, I noticed that the `<article>` element and its descendants were modified by the same `line-height` anyway, so I kept the declaration and added the class to elements that were previously `<article>` elements.
 
| Using `page-content__prose`      | Using new `page-content__paragraph` |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/17969963/236219559-09d8e56a-60df-4c7e-98eb-ce93a34f3402.png)      | ![image](https://user-images.githubusercontent.com/17969963/236225558-3870ed49-d45f-4ec1-a73f-c47b126f1f60.png)       |

Reply:
Based on the table, did you intend to change the name from `container--mod` to `page-content__paragraph` ?